sys/fs/constfs/Makefile

```diff
@@ -1,2 +1,6 @@
 MODULE=constfs
+
+# Required for strnlen in string.h, when building with -std=c99
```

Comment:
Since we are building with `-std=c11` now, do we even still need this?

Reply:
Yes, if you comment these lines out the build will fail, at least on my machine (`arm-none-eabi-gcc (GNU Arm Embedded Toolchain 9-2020-q2-update) 9.3.1 20200408 (release)`)